spec/models/activity_spec.rb

```diff
@@ -147,4 +147,26 @@
       end
     end
   end
+
+
+  describe '#entity_uris' do
+    include_context 'provenance queries'
+    include_context 'entities query'
+    # See spec/factories/krikri_activities.rb
+    # generator_uri is the URI of the harvest activity
+    # generator_uri matches what Krikri::Activity will construct as the
+    # uri, given its value of #rdf_subject, in #aggregations_as_json
+    # See 'provenance queries' shared context.  
+    let(:generator_uri) do
```

Comment:
Same comment as for [`aggregation_entity_behavior_spec.rb`](https://github.com/dpla/KriKri/pull/131/files#r27392504), without @no-reply's feedback. I wonder if `krikri_activity.rdf_subject` will work here.